doc/api/url.markdown

```diff
@@ -64,6 +64,11 @@ string will not be in the parsed object. Examples are shown for the URL
 
     Example: `'#hash'`
 
+The following characters and empty spaces will be automatically escaped in the
+properties of URL objects:
+
+    < > " ` \r \n \t { } | \ ^ ~ '
```

Comment:
Is the tilde still done? I thought it was removed in https://github.com/nodejs/node/commit/881ef7cc5fc897cca2f2f0a512e59111ff5a6cd8?